cli/src/test/kotlin/ExtensionsTest.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.should
+import io.kotest.matchers.string.endWith
+
+import org.eclipse.apoapsis.ortserver.cli.ensureTrailingSlash
+
+class ExtensionsTest : StringSpec({
+    "ensureTrailingSlash" should {
+        "add a trailing slash to a string if missing" {
+            "http://localhost:8080".ensureTrailingSlash() should endWith("/")
+        }
+
+        "not add a trailing slash to a string if already present" {
+            "http://localhost:8080/".ensureTrailingSlash() should endWith("/")
```

Comment:
This does not really verify that not two slashes are at the end.

Reply:
Agreed, better check the whole expected string outcome. And you also do not check whether this new function is called when constructing the `OrtServerOptions`. Is there an easy way to test the clikt option processing?

Reply:
> Is there an easy way to test the clikt option processing?

There isn't.

Reply:
Maybe things have changed with the resolution of https://github.com/ajalt/clikt/issues/449?